lambdas/pkgpush/index.py

```diff
@@ -172,6 +179,51 @@
 logger = get_quilt_logger()
 
 
+def calculate_pkg_hashes(s3_client, pkg):
+    entries = []
+    for lk, entry in pkg.walk():
+        if entry.hash is not None:
+            continue
+        if entry.size > S3_HASH_LAMBDA_MAX_FILE_SIZE:
+            raise FileTooLargeForHashing(lk)
+
+        entries.append(entry)
```

Comment:
where do we actually use `entries`? i see `data['entries']` below so maybe this is some context magic but i find it unclear.